### PR TITLE
[CSS Gradient] -> [CSS Frameworks]

### PR DESCRIPTION
**Pull request by varghesejose2020**
# CSS Gradient

CSS Gradient is a little website and free tool that lets you create a gradient background for websites

Link: https://cssgradient.io/gradient-backgrounds/

#### Checklist:

- [x] I have performed a self-review of submitted resource and its follows the guidelines of the project.

### CODE REVIEW

**Review by Alaev — CHANGES_REQUESTED** [review]
we have gradient on our list

https://www.grabient.com/

**Comment by varghesejose2020** [comment]
@Alaev MY PR IS  https://cssgradient.io/gradient-backgrounds/

File: readme.md [hunk 1]
```diff
@@ -543,6 +543,7 @@
 | [Halfmoon UI](https://www.gethalfmoon.com/)| A responsive and lightweight framework, designed for quickly building beautiful dashboards and product pages. |
 | [Metro 4](https://metroui.org.ua/index.html)| Create your site quickly and effectively with Metro 4. impressive components library built on html, css, javascript. |
 | [css-doodle](https://css-doodle.com/) | A web component for drawing patterns with CSS |
+| [CSS Gradient](https://cssgradient.io/gradient-backgrounds/) | CSS Gradient is a little website and free tool that lets you create a gradient background for websites |
```
Comment (Alaev) [inline readme.md:546]: you can add cssgradient as its own resource with the link `https://cssgradient.io/` 
```suggestion
| [CSS Gradient](https://cssgradient.io) | CSS Gradient is a little website and free tool that lets you create a gradient background for websites |
```

**Review by Alaev — CHANGES_REQUESTED** [review]

**Comment by Alaev** [comment]
> @Alaev MY PR IS https://cssgradient.io/gradient-backgrounds/

yep I saw your pr. however if you look at the link you can see that the website and the link I sent you its the same resource as its pulls the background from my link :)

https://cssgradient.io/gradient-backgrounds/
Screenshot from 2021-04-05 14-56-27

https://www.grabient.com/
Screenshot from 2021-04-05 14-55-55

Do you see it?

**Comment by varghesejose2020** [comment]
:+1:

**Comment by Alaev** [comment]
@varghesejose2020 you can change the link as I suggested and we can add it to the list as its own resource if you like.

**Comment by varghesejose2020** [comment]
> @varghesejose2020 you can change the link as I suggested and we can add it to the list as its own resource if you like.

changed

**Review by Alaev — APPROVED** [review]

**Comment by Alaev** [comment]
@varghesejose2020 Great, can you just move it to Colors section?

**Comment by Alaev** [comment]
Never mind, Closing as we have it in our list under Colors.

**Pull request closed without merging**